tools/lib.sh

```diff
@@ -7,18 +7,26 @@
 #  LICENSE file in the root directory of this source tree. An additional grant
 #  of patent rights can be found in the PATENTS file in the same directory.
 
+ORACLE_RELEASE=/etc/oracle-release
 SYSTEM_RELEASE=/etc/system-release
 UBUNTU_RELEASE=/etc/lsb-release
 
 function platform() {
   local  __out=$1
-  if [[ -n `grep -o "CentOS" $SYSTEM_RELEASE 2>/dev/null` ]]; then
+  if [[ -f "$ORACLE_RELEASE" ]]; then
+    FAMILY="redhat"
```

Comment:
@javuto so if I declare these `$FAMILY` variables within a call to `platform $OS` I should be ok?